cutscene_engine/cutscene_scripts/DialogueStarter.gd

```diff
@@ -0,0 +1,41 @@
+# Initializes the map's pawns and emits signals so the
+# Game node can start encounters
+# Processes in pause mode
+extends Node
+class_name DialogueStarter
+
+onready var dialogue_box = $DialogueInterface/DialogueBox
+#onready var trigger_box_setter = $TriggerBoxSetter
+
+var trigger_counter = 0
+
+export var key = ""
+var dialogue_key
```

Comment:
Can you explain why these need to be global? Looks like they could be passed as function args when they are needed.